e-shop-website-with-ilb-ase/metadata.json

```diff
@@ -5,6 +5,6 @@
 	"description": "An App Service Environment is a Premium service plan option of Azure App Service that provides a fully isolated and dedicated environment for securely running Azure App Service apps at high scale, including Web Apps, Mobile Apps, and API Apps.",
 	"summary": "This is a sample for LoB solution with App Service Environment (ASE).",
 	"githubUsername": "tylerlu",
-	"dateUpdated": "2018-07-02"
+	"dateUpdated": "2020-09-02",
+  	"environments": ["AzureCloud"]
```

Comment:
we have param files for USGov, but then are excluding it here - if this is supported in USGov, remove the environments property, if it's not, remove the .us. param files...